.github/workflows/labeler.yml

```diff
@@ -0,0 +1,16 @@
+name: "Pull Request Labeler"
```

Comment:
Maybe change the name of the file too? I'd like to have a e.g. `to-triage` label for issues.

Reply:
Happy to rename things.

I'm not going to add the other label/confit/workflow until we discuss how that process will work and how/when we're considering triage complete. (That's probably for another PR.)

Reply:
Tweaked.

Reply:
Yeah, not asking for a new workflow here, just giving the reason to use a more specific name.